internal/statestore/redis.go

```diff
@@ -584,7 +546,14 @@ func handleConnectionClose(conn *redis.Conn) {
 }
 
 // TODO: add cache the backoff object
-func (rb *redisBackend) createBackoffStrat() backoff.BackOff {
+func (rb *redisBackend) createConstantBackoffStrat() backoff.BackOff {
```

Comment:
nit - suggest newConstantBackoffStrategy() as the name (drop create?) 

Since this does not have any errors.. do you even need a helper? just use
backoff.BackOff(backoff.NewConstantBackOff(rb.cfg.GetDuration("backoff.initialInterval")))
?